systemtest/juelich_tests/test_11/runtest_juelich_11.py

```diff
@@ -1,78 +1,45 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 import logging
-import numpy as np
 import os
 import subprocess
+import sys
 from os import path
-from sys import argv, exit
-
+from shutil import copyfile
+utestdir = os.path.abspath(os.path.dirname(os.path.dirname(sys.path[0])))
+sys.path.append(utestdir)
+from JPSRunTest import JPSRunTestDriver
+from utils import *
 #=========================
 testnr = 11
 #========================
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
+PATH = os.path.dirname(os.path.abspath(__file__))
 
-SUCCESS = 0
-FAILURE = 1
-#--------------------------------------------------------
-
-logfile="log_test_%d.txt"%testnr
-f=open(logfile, "w")
-f.close()
-logging.basicConfig(filename=logfile, level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
+def get_evak_time(inifile, trajfile):
+    fps, N, traj = parse_file(trajfile)
+    etime =  (max( traj[:, 1] ) - min(traj[:, 1])) / float(fps)
+    return etime
 
+def call_test(num, suffix):
+    logging.info("copy master_ini_%s.xml"%suffix)
+    logging.info("copy geometry/geometry_test11_%s.xml"%suffix,)
+    copyfile("%s/master_ini_%s.xml"%(PATH, suffix), "%s/master_ini.xml"%PATH)
+    copyfile("%s/geometry/geometry_test11_%s.xml"%(PATH, suffix), "%s/geometry.xml"%PATH)
```

Comment:
Why are these files copied? Do we need them somewhere else? Isn't this done by make_ini?

Reply:
these are copied because this test is composed of two subtests that cover two cases:
- Subrooms
- Rooms

Idially makeini, would glob over geometries, but I don't know how to do neither I would like to do it, knowing that makeini has no future.